### PR TITLE
Update documentation after pilot program with latest details.

### PR DESCRIPTION
**Pull request by sarahboyce**
- updated the session duration to 8 weeks
- updated the time commitments to be closer to what we experienced
- removed the old code of conduct file as I think there has been sufficient time for the linking to the new file
- removed our program overview file as it is a bit misleading around how the program currently is. I think we create something a bit clearer when we have done a few iterations of the program
- removed the "ticket grooming" point as it is not a hard requirement and puts people off

### CODE REVIEW

File: navigators.md [hunk 1]
```diff
@@ -72,7 +67,7 @@ there to help guide them by sharing your methodologies, resources, tips and expe
 
 - Have some fun and learn together!
 
-As a rough guide, being a Navigator should be a 2-4 hour per week commitment for 3 months. 
+As a rough guide, being a Navigator should be roughly a 2 hour per week commitment for 8-weeks.
```
Comment (sabderemane) [inline navigators.md:70]:
```suggestion
As a rough guide, being a Navigator should be roughly a 2-hour per week commitment for 8 weeks. 
```
Not sure, but I guess this should be correct

File: program-overview.md [hunk 2]
```diff
@@ -1,18 +0,0 @@
-### Vision :rocket: 
-Djangonaut Space is a community where open source contributing newcomers, specifically people from historically excluded groups, are welcome and supported to get on their feet to contribute to the Django project.
```
Comment (sabderemane) [inline program-overview.md:2]: Same, this file is not used via a link on the homepage or landing page?

**Review by sabderemane — COMMENTED** [review]
Nice work! I just had few comments / asking

File: program-overview.md (hunk 2 above)
Comment (sarahboyce) [reply program-overview.md:2]: Yes this is not used 👍

File: conduct.md [hunk 3]
```diff
@@ -1,3 +0,0 @@
-### The [Code of Conduct](CODE_OF_CONDUCT.md) has moved!
-
-Please view our [Code of Conduct here](CODE_OF_CONDUCT.md). :heart:
```
[the remark this answers is not in the document]
Comment (sarahboyce) [reply conduct.md:3]: This was once used, but not anymore 👍 
I think it's enough time that we can clean it up

**Review by sabderemane — APPROVED** [review]

**Pull request merged**